### PR TITLE
Upgrade to Vaadin 8.7.2

### PR DESCRIPTION
**Pull request by ZheSun88**

### CODE REVIEW

File: start-site/src/main/resources/application.yml [hunk 1]
```diff
@@ -1292,4 +1292,4 @@ spring:
     restart:
       additional_paths:
         - start-client/build/
-      additional_exclude: "**/*.js,**/*.css"
\ No newline at end of file
+      additional_exclude: "**/*.js,**/*.css"
```
Comment (snicoll) [inline start-site/src/main/resources/application.yml]: Can you please remove that unrelated change?

Comment (ZheSun88) [reply start-site/src/main/resources/application.yml]: Remove the new line at the end.

Comment (ZheSun88) [reply start-site/src/main/resources/application.yml]: Not sure where this comes, maybe it comes from the old fork. If it is still an issue, I will redo the fork and make a new PR.

**Comment by snicoll** [comment]
>  If it is still an issue, I will redo the fork and make a new PR.

I am afraid it is and I wouldn't bother for a one line change so I've done it in https://github.com/spring-io/start.spring.io/commit/a4942a74fd60697c0b339c7bd912b898f9d1df0a - Thanks for letting us know about the upgrade in any case!
